irc/errors.go

```diff
@@ -9,23 +9,24 @@ import "errors"
 
 // Runtime Errors
 var (
-	errAccountAlreadyRegistered       = errors.New("Account already exists")
-	errAccountAlreadyVerified         = errors.New("Account is already verified")
+	errAccountAlreadyRegistered       = errors.New(`Account already exists`)
+	errAccountAlreadyVerified         = errors.New(`Account is already verified`)
 	errAccountCantDropPrimaryNick     = errors.New("Can't unreserve primary nickname")
 	errAccountCreation                = errors.New("Account could not be created")
 	errAccountCredUpdate              = errors.New("Could not update password hash to new method")
 	errAccountDoesNotExist            = errors.New("Account does not exist")
 	errAccountInvalidCredentials      = errors.New("Invalid account credentials")
-	errAccountBadPassphrase           = errors.New("Passphrase contains forbidden characters or is otherwise invalid")
+	errAccountBadPassphrase           = errors.New(`Passphrase contains forbidden characters or is otherwise invalid`)
 	errAccountNickReservationFailed   = errors.New("Could not (un)reserve nick")
 	errAccountNotLoggedIn             = errors.New("You're not logged into an account")
 	errAccountTooManyNicks            = errors.New("Account has too many reserved nicks")
 	errAccountUnverified              = errors.New("Account is not yet verified")
 	errAccountVerificationFailed      = errors.New("Account verification failed")
 	errAccountVerificationInvalidCode = errors.New("Invalid account verification code")
 	errAccountUpdateFailed            = errors.New("Error while updating your account information")
+	errAccountMustHoldNick            = errors.New(`You must hold that nickname in order to register it`)
```

Comment:
Just curious, what's with the backticks on a few (seemingly) random error strings here?

Reply:
...not yet, but I'll add that to the TL grabber! (it only does it for specific files right now, I'll make it include errors.go as well)